src/python/gem5/components/boards/se_binary_workload.py

```diff
@@ -23,52 +23,49 @@
 # THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
```

Comment:
I suggest not committing whitespace-only changes unless you're updating the style to be more compliant.